### DISCUSSION TITLE
How to get remaining time to live of a key that has a timeout ?

### DISCUSSION THREAD

**Discussion by hnimminh** [nutsdb/nutsdb#287] [Q&A] [answered]
@xujiajun
Is there a way to get TTL of key memory mode? equivalent to TTL in Redis.
Example:
GetTTL(BUCKET, KEY)

**Comment by xujiajun** [comment, marked as answer]
@hnimminh Sorry for taking so long to reply to you. You can see the reply to this issue https://github.com/nutsdb/nutsdb/issues/304
